mne/fixes.py

```diff
@@ -1219,3 +1219,285 @@ def __repr__(self):
     # __getstate__ and __setstate__ are omitted because they only contain
     # conditionals that are not satisfied by our objects (e.g.,
     # ``if type(self).__module__.startswith('sklearn.')``.
+
+
+###############################################################################
+# Copied from sklearn to simplify code paths
+
+def empirical_covariance(X, assume_centered=False):
```

Comment:
why this big diff?

Reply:
We don't require `sklearn` for emprical, but I unified the code path to always use `sklearn`-like estimators (whereas before we manually implemented the `np.cov` equivalent one) as it simplifies a lot of code. Now we need to copy in the `sklearn` empirical cov estimator to continue not to require `sklearn` when doing `method='empirical'`.